snippets/language-ruby.cson

```diff
@@ -15,6 +15,9 @@
   'case … end':
     'prefix': 'case'
     'body': 'case ${1:object}\nwhen ${2:condition}\n\t$0\nend'
+  'begin … rescue … end':
+    'prefix': 'bre'
```

Comment:
I have a shortcut for this locally on my machine that uses "beg" as the prefix. "beg" seems more common with the other snippets (ie: "if" "cla", etc.). Sublime and I think textmate also used "beg" I believe. Thoughts on using "beg" for consistency?